Structure_Engine/Query/TorsionalConstant.cs

```diff
@@ -406,6 +497,118 @@ private static double AlphaLJunction(double tw, double tf, double r)
         }
 
         /***************************************************/
+
+        [Description("Empirical formula used to correct the torsional constant with enhancement from a tapered T-junction such as a taper flange I-Section. Taken from Johnston & El Darwish, 1964")]
+        [Input("tw", "Web thickness.", typeof(Length))]
+        [Input("tf", "Mean flange thickness.", typeof(Length))]
+        [Input("r", "Root radius.", typeof(Length))]
+        [Input("t2", "Thickness of flange at intersection with near face of web.", typeof(Length))]
+        [Input("s", "Flange taper angle.", typeof(Length))]
+        private static double AlphaTaperTJunction(double tw, double tf, double r, double t2, double s)
+        {
+            if (0.2 > r / t2 || r / t2 > 1.0)
+                Reflection.Compute.RecordWarning("Calculation of alpha term of torsional constant may not be accurate because ratio of root radius to flange thickness is out of the applicable range.");
+
+            if (0.2 > tw / t2 || tw / t2 > 1.0)
+                Reflection.Compute.RecordWarning("Calculation of alpha term of torsional constant may not be accurate because ratio of web thickness to flange thickness is out of the applicable range.");
+
+            double alpha0 = AlphaTJunction(tw, t2, r);
+
+            //Equation 28
+            double alpha16 = -0.0836 + 0.2536 * tw / t2 + 0.1268 * r / t2 - 0.0806 * tw * r / Math.Pow(t2, 2) - 0.0858 * Math.Pow(tw / t2, 2); 
+            //Interpolate alpha between parallel flange and slope of 1 to 6:
+            return alpha0 + alpha16 * s / Math.Atan(0.1667);
```

Comment:
Think this interpolation is a bit off. Think it should be the following:
```suggestion
            return alpha0 + (alpha16 - alpha0) * s / Math.Atan(0.1667);
```